tests/ha/hawk_gui.pm

```diff
@@ -105,14 +104,15 @@ sub run {
     }
     if ($loop_count < 0) {
         record_info("$browser failed", "Test with browser [$browser] could not be completed in 30 minutes", result => 'softfail');
-        script_run "docker container kill test";
+        script_run "podman container kill test";
     }
 
     save_screenshot;
 
     assert_screen "generic-desktop";
     barrier_wait("HAWK_GUI_CPU_TEST_FINISH_$cluster_name");
 
+    script_run("exit;");
```

Comment:
is it to `exit` from `become_root`? Is there not a proper API like `leave_root` ?